### PR TITLE
Add test for SymbolHelper::IsMatchingDebugInfoFile

### PR DESCRIPTION
**Pull request by beckerhe**

### CODE REVIEW

File: src/OrbitCore/SymbolHelper.h [hunk 1]
```diff
@@ -33,6 +33,7 @@ class SymbolHelper {
 
   [[nodiscard]] fs::path GenerateCachedFileName(const fs::path& file_path) const;
 
+  [[nodiscard]] static bool IsMatchingDebugInfoFile(const fs::path& file_path, uint32_t checksum);
```
Comment (dimitry-) [inline src/OrbitCore/SymbolHelper.h:36]: probably did not mean to make this static

**Review by dimitry- — APPROVED** [review]
otherwise lgtm

File: src/OrbitCore/SymbolHelper.h (hunk 1 above)
Comment (beckerhe) [reply src/OrbitCore/SymbolHelper.h:36]: Hmm why not? It does not need access to any member variables. I could also make it a free function if you prefer that?

Comment (dimitry-) [reply src/OrbitCore/SymbolHelper.h:36]: nevermind, missed the part where you moved it to SymbolHelper class.

**Pull request merged**